tests/integration/route_test.py

```diff
@@ -1834,6 +1834,47 @@ def test_absent_route_with_invalid_empty_destination(static_eth1_with_routes):
         )
 
 
+@pytest.mark.tier1
+def test_add_ip_routes_reapply(eth1_static_ip):
```

Comment:
Reapply is NM terminology, please use `test_preserve_unmanaged_routes()`

Reply:
I changed it to test_preserve_unmanaged_routes.